pkg/pipeline/encode/encode_prom_test.go

```diff
@@ -485,8 +524,8 @@ func buildFlow() config.GenericMap {
 }
 
 func hundredFlows() []config.GenericMap {
-	flows := make([]config.GenericMap, 100)
-	for i := 0; i < 100; i++ {
+	flows := make([]config.GenericMap, 1000)
+	for i := 0; i < 1000; i++ {
```

Comment:
rename the function above then :nerd_face:

Reply:
done!